v1/feed.yaml

```diff
@@ -71,18 +71,13 @@ paths:
                   - views: /.+/
                     rank: /.+/
                     title: /.+/
-                    pageid: /.+/
-                    normalizedtitle: /.+/
               random:
                 items:
                   - title: /.+/
               news:
                 - story: /.+/
                   links:
-                    - pageid: /.+/
-                      ns: /.+/
-                      normalizedtitle: /.+/
```

Comment:
Yeah, we do.  Good point, probably better to keep checking for it.  I'm going to do a little more testing and maybe withdraw this PR if we can get back with the spec as-is.